src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,15 @@
+$day-size: 100px;
+$row-days: 7;
+$days-gap: 1px;
+$padings: 10px;
+
+$calendar-width: $row-days * $day-size + $padings * 2;
+
```

Comment:
i think you can use function in separate file for calculate this number

Reply:
this calculation used only one time, so it's not required to create separate function for it